iothub/service/tests/ExceptionHandlingHelperTests.cs

```diff
@@ -48,17 +69,17 @@ public async Task GetExceptionCodeAsync_ContentAndHeadersMisMatch_InvalidErrorCo
             httpResponseMessage.Headers.Add(HttpErrorCodeName, "DummyErrorCode");
 
             // act
-            ErrorCode errorCode = await ExceptionHandlingHelper.GetExceptionCodeAsync(httpResponseMessage);
+            IotHubErrorCode errorCode = await ExceptionHandlingHelper.GetIotHubErrorCodeAsync(httpResponseMessage);
 
             // assert
-            Assert.AreEqual(ErrorCode.InvalidErrorCode, errorCode);
+            errorCode.Should().Be(IotHubErrorCode.Unknown);
         }
 
         [TestMethod]
-        public async Task GetExceptionCodeAsync_NoContentErrorCode_InvalidErrorCode()
+        public async Task GetExceptionCodeAsync_NoContentErrorCode_UnknownErrorCode()
         {
             // arrange
-            var httpResponseMessage = new HttpResponseMessage(System.Net.HttpStatusCode.BadRequest);
+            var httpResponseMessage = new HttpResponseMessage(HttpStatusCode.BadRequest);
             var exceptionResult = new IoTHubExceptionResult
             {
                 Message = ""
```

Comment:
Should still be a string with content, just not in a format we expect so we can't parse it.